src/custom/components/TransactionConfirmationModal/TransactionConfirmationModalMod.tsx

```diff
@@ -303,6 +305,25 @@ export function L2Content({
 
   const info = getChainInfo(chainId)
 
+  const displayIcon = useMemo(() => {
+    if (isLimitOrderSubmit) {
+      return <AnimatedConfirmation />
+    } else if (!confirmed) {
+      return <CustomLightSpinner src={Circle} alt="loader" size={inline ? '40px' : '90px'} />
```

Comment:
Not very good to add PX in the JSX, you should pass the property inline to the component, not the size (I think)